crates/process_info/src/linux.rs

```diff
@@ -0,0 +1,35 @@
+#![cfg(target_os = "linux")]
+use super::ProcessInfo;
+
+use std::{fs, path::PathBuf};
+
+impl ProcessInfo {
+    pub fn load(pid: u32) -> Option<Self> {
```

Comment:
Honestly, somewhat confused that a linux version is relying on procfs parsing.
I understand that this is a somewhat standard in modern Linux distros, but we do have `libc` at our disposal and seem to do something very similar in `macos.rs` already, can we have something similar for Linux part too?

Reply:
I'm using macos right know so I don't know how to make such a big change without being able to test properly.

Reply:
There are so many ways including Docker, but sure, let's keep it going for now, and see if terminal on Linux misbehaves on merge.